crates/fluvio-controlplane-metadata/src/spu/status.rs

```diff
@@ -34,6 +34,7 @@ impl SpuStatus {
             SpuStatusResolution::Online => "online",
             SpuStatusResolution::Offline => "offline",
             SpuStatusResolution::Init => "Init",
+            SpuStatusResolution::OutOfStorage => "out of storage",
```

Comment:
See overall comment.  No change here since this is for SPU health

Reply:
btw, should we create a new status for partition with the same name (OutOfStorage) or just use offline there?

Reply:
Yes, add `OufOfStorage` for partition status

Reply:
fixed